main/src/com/google/refine/commands/project/ExportRowsCommand.java

```diff
@@ -100,6 +100,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
                 contentType = exporter.getContentType();
             }
             response.setHeader("Content-Type", contentType);
```

Comment:
Did I miss something?  but L102 still has Content-Type, even though you mentioned and changed to Content-Disposition ... was L102 missed as a deleted line?

Reply:
The removed content type of `application/x-unknown` was in exporters.js. We still want to set the real content type e.g. `text/html`